custom-addons/esb/account_form/reports/tax_invoice_form.xml

```diff
@@ -146,6 +146,7 @@
                         <!-- <t t-set="total_discount" t-value="0"/> -->
                         <t t-set="current_subtotal" t-value="0"/>
                         <t t-foreach="o.invoice_line_ids" t-as="line">
+                            <t t-set="price" t-value="line._compute_price_unit_in_vat(line)"/>
```

Comment:
IMO, I think it is not necessary to pass the `line` in the function `_compute_price_unit_in_vat()` because you pass `line` as `self`.